src/main/java/tutorspet/storage/JsonAdaptedModuleClass.java

```diff
@@ -94,10 +95,19 @@ private boolean hasDuplicateLessons(List<Lesson> lessons, Lesson lessonToCheck)
         return lessons.stream().anyMatch(lessonToCheck::isSameLesson);
     }
 
+    /**
+     * Checks if a {@code lesson} timing overlaps with another {@code Lesson} in the list.
+     * Overlapping lessons are detected by calling the {@code isOverlapLesson} method in {@code Lesson}.
```

Comment:
Perhaps it might be good to specify class here?
```suggestion
     * Overlapping lessons are detected by calling the {@code isOverlapLesson} method in {@code Lesson} class.
```